libs/wingc/src/jsify.rs

```diff
@@ -411,15 +416,31 @@ impl<'a> JSifier<'a> {
 						.join("\n")
 				)
 			}
+      ExprKind::JsonMapLiteral { fields, type_ } => {
+        let f = fields
+          .iter()
+          .map(|(key, expr)| format!("[ \"{}\", {} ]", key, self.jsify_expression(expr, phase)))
+          .collect::<Vec<String>>()
+          .join(", ");
+        
+        match type_ {
+          TypeAnnotation::MutJson => format!("new Map([{}])", f),
```

Comment:
Why are we using a `Map` here as the underlying struct instead of a plain-old javascript object?